pkg/common/morpc/backend.go

```diff
@@ -1023,10 +1024,12 @@ func (s *stream) done(
 	}
 
 	s.lastReceivedSequence = message.streamSequence
-	select {
-	case s.c <- response:
-	case <-ctx.Done():
-	}
+	moprobe.WithRegion(ctx, moprobe.RPCStreamReceive, func() {
+		select {
```

Comment:
I think not a problem, but never used select in a function then trace.

What do you want to trace?    I would trace the Send and Recv function.   What is the meaning of done in terms of rpc send/recv?   But this is what you know better.

Reply:
I just want to see how long it takes to send the message to the channel, because if the channel is consumed in cn, if the consumption blocks, this place will also block, the socket read will also be blocked, then eventually the dn write will also be blocked